AssettoServer/Network/Packets/Protocol.cs

```diff
@@ -0,0 +1,92 @@
+﻿namespace AssettoServer.Network.Packets
+{
+    public enum ACServerProtocol : byte
+    {
+        P2PUpdate              = 0x0D,
+        MandatoryPitUpdate     = 0x0E,
+        Handshake              = 0x3B,
+        WrongPassword          = 0x3C,
+        RequestConnectionUdp   = 0x3D,  // Not supported in new clients anymore
```

Comment:
I'm a bit confused by this comment because 0x3D is used during the handshake

Reply:
ignore it, i made it as a mental note to check something, but forgot to remove it